packages/legacy/core/App/navigators/TabStack.tsx

```diff
@@ -21,7 +21,7 @@ import HomeStack from './HomeStack'
 const TabStack: React.FC = () => {
   const { fontScale } = useWindowDimensions()
   const [{ useNotifications }] = useServices([TOKENS.NOTIFICATIONS])
-  const notifications = useNotifications()
+  const notifications = useNotifications({})
```

Comment:
The way useNotifications is being used could probably be refactored here. The way it is currently implemented in this PR, notifications on the home screen won't be seen for OpenID creds.

Reply:
The is a subsequent PR to support displaying OPenID in the home notifications and in credentials List, it not part of this PR. We discussed this approach with Cleo and Jason in our Friday dev calls and we agreed to pass an empty object for now since react hooks doesn't support undefined